ta/secure_key_services/src/processing_ec.c

```diff
@@ -1274,6 +1275,15 @@ uint32_t generate_ec_keys(struct sks_attribute_head *proc_params,
 
 	TEE_InitValueAttribute(&tee_key_attr[0], TEE_ATTR_ECC_CURVE,
 				tee_curve, 0);
+	tee_count++;
+
+	/* pass the label to the private exponent */
+	rv = get_attribute_ptr(*pub_head, SKS_CKA_LABEL, &b_ptr, &b_size);
+	if (rv == SKS_OK) {
```

Comment:
um, this must not be done for every label, only those prefixed with SE_. let me fix.